frontend/app/src/locales/en.json

```diff
@@ -102,7 +102,19 @@
     },
     "title": "A supported asset update is available",
     "total_changes": "This update contains {changes} asset changes.",
-    "up_to_version": "Only update up to version"
+    "up_to_version": "Only update up to version",
+    "restore": {
+      "title": "Restore defaults assets database",
+      "action": "Restore",
+      "delete_confirmation": {
+        "title": "Restore defaults assets database",
+        "message": "Do you really want to restore your local copy of assets? This will keep the assets that you manually added."
```

Comment:
```suggestion
        "message": "Do you really want to restore your local assets database? This will keep the assets that you manually added and were not known to rotki before and reset all others to their original state."
```